pyproject.toml

```diff
@@ -8,9 +8,20 @@ authors = []
 python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
+<<<<<<< HEAD
+=======
+requests = "^2.24.0"
+pytest = "^6.2.1"
+selenium = "^3.3.0"
+pytest-mock = "^3.5.1"
```

Comment:
It's better to put these test dependencies in the dev-dependencies section, so that they can be skipped when setting up a server.

(They could have been added with e.g. `poetry add pytest --dev`)